pkgs/build-support/agda/default.nix

```diff
@@ -59,7 +59,12 @@ let
         installPhase = if installPhase != null then installPhase else ''
           runHook preInstall
           mkdir -p $out
-          find \( -name '*.agda' -or -name '*.agdai' -or -name '*.agda-lib' \) -exec cp -p --parents -t "$out" {} +
+          find \( -name '*.agda' \
```

Comment:
You could put the patterns in a list of strings and use `builtins.map` and `builtins.concatStringsSep` to create this command.

Reply:
sure